cmd/bbolt/main.go

```diff
@@ -7,26 +7,12 @@ import (
 )
 
 var (
-	// ErrUsage is returned when a usage message was printed and the process
-	// should simply exit with an error.
-	ErrUsage = errors.New("usage")
-
-	// ErrUnknownCommand is returned when a CLI command is not specified.
-	ErrUnknownCommand = errors.New("unknown command")
-
 	// ErrPathRequired is returned when the path to a Bolt database is not specified.
 	ErrPathRequired = errors.New("path required")
```

Comment:
Should we move this error definition to `cmd/bbolt/command_get.go`?

Reply:
lgtm 👍🏽